crates/fj-viewer/src/input/handler.rs

```diff
@@ -134,15 +151,21 @@ impl Handler {
 }
 
 #[derive(Default)]
+/// Hight level application state container.
```

Comment:
Two notes.

First, a typo:
```suggestion
/// High level application state container.
```

Second, I don't think that description is accurate. This struct contains the actions the user is taking this frame. For example, `toggle_model` is true, if there was input that should result in the model display state being toggled, but it doesn't reflect whether the model is currently being shown or not.

This will affect application state, but it isn't really application state. (Which is why it's in the `input` module.)

Reply:
Resolved by: https://github.com/hannobraun/Fornjot/pull/487/commits/b5f9d360e3b83f96cd9f5c575fd85853cc22031d